axiom/auth/auth.go

```diff
@@ -88,12 +87,13 @@ func Login(ctx context.Context, baseURL string, loginFunc LoginFunc) (string, er
 		return "", err
 	}
 
-	state, err := uuid.NewUUID()
-	if err != nil {
+	stateBytes := make([]byte, 16)
+	if _, err = rand.Read(stateBytes); err != nil {
 		return "", err
 	}
+	state := hex.EncodeToString(stateBytes)
```

Comment:
Missing some context here, why are we moving to a random hex string?

Reply:
I took some inspiration from different OAuth2 client-side implementations that utilise `state`. While my previous choice of UUID provides sufficient entropy, that approach wasn't used a lot in the wild.

Also the new approach makes it a bit more explicit where entropy is derived from as compared to the UUID approach where you actually need some insight into what an UUID is and how it works.